FeedAPIChallenge/Feed API/RemoteImagesLoader.swift

```
@@ -0,0 +1,17 @@
+//
+//  RemoteImagesLoader.swift
+//  FeedAPIChallenge
+//
+//  Created by Ivan Ornes on 10/1/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal struct RemoteImagesLoader {
+	
+	internal static func getImages(_ data: Data) throws -> [FeedImage] {
+		guard let dictionary = try JSONSerialization.jsonObject(with: data, options: .allowFragments) as? [String: Any] else { return [] }
+		return (dictionary["items"] as? [[String: Any]] ?? []).compactMap(RemoteImage.init).map { $0.feedImage }
```

`dictionary["items"] as? [[String: Any]]` is part of the data validation/extraction process. 

If you can't access the items as `[[String: Any]]` then the data should be considered invalid and you should throw an error.

Include this check in the guard statement.